l10n_it_abicab/models/abicab.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2015 Associazione Odoo Italia (<http://www.odoo-italia.org>)
+# Copyright 2016 Davide Corio (Abstract)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields, api
+
+
+class ResBank(models.Model):
+
+    _inherit = "res.bank"
+
+    abi = fields.Char(size=5, string='ABI')
+    cab = fields.Char(size=5, string='CAB')
+
+
+class ResPartnerBank(models.Model):
+
+    _inherit = "res.partner.bank"
+
+    bank_abi = fields.Char(size=5, string='ABI')
+    bank_cab = fields.Char(size=5, string='CAB')
```

Comment:
In customer form > accounting > bank accounts
when you create a bank account, selecting a bank with ABI and CAB filled, ABI and CAB of the account are not filled.

What do you think about making these 2 fields related?

Reply:
I don't get it.
Steps:
Accounting→Configuration→Bank Accounts→Create
Fill in "Bank Account Name" then select "Create and Edit..." in "Bank Account" field.
In the new form select a bank that has ABI and CAB set.
After that ABI and CAB fields below "Bank" get filled in the form.

Reply:
Ooops, sorry... :sweat_smile: :stuck_out_tongue_closed_eyes: 

Done!